modules/oe_bootstrap_theme_paragraphs/src/EventSubscriber/IconOptionsSubscriber.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_bootstrap_theme_paragraphs\EventSubscriber;
+
+use Drupal\oe_paragraphs\Event\IconOptionsEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Provides options for the icon field.
+ */
+class IconOptionsSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents(): array {
+    return [
+      IconOptionsEvent::class => ['getIconOptions', -1],
+    ];
+  }
+
+  /**
+   * Gets the icon options.
+   *
+   * @param \Drupal\custom_events\Event\IconOptionsEvent $event
+   *   Allowed format event object.
+   */
+  public function getIconOptions(IconOptionsEvent $event): void {
+    $event->setIconOptions([
+      'arrow-down' => 'Arrow down',
+      'box-arrow-up' => 'External',
+      'arrow-up' => 'Arrow up',
+      'book' => 'Book',
+      'camera' => 'Camera',
+      'check' => 'Check',
+      'download' => 'Download',
+      'currency-euro' => 'Euro',
+      'facebook' => 'Facebook',
+      'file' => 'File',
+      'image' => 'Image',
+      'info' => 'Info',
+      'linkedin' => 'LinkedIn',
+      'files' => 'Multiple files',
+      'rss' => 'RSS',
+      'search' => 'Search',
+      'share' => 'Share',
+      'twitter' => 'Twitter',
+      'person-video' => 'Video',
```

Comment:
For some reason, "Video" icon does not work for me locally.
Are we sure that all the icons exist in BCL?

Reply:
Right, now we don't have the person-video on file bootstrap-icons.svg. At the time when we did it it was there. So we must check if we can add that icon now. @escuriola we must check now at the file what we have available. At the time I did this, I checked at the bootstrap icon site, because the FRONT team told us that the all icons from [here](https://icons.getbootstrap.com/) were on the file.